code/modules/library/skill_learning/skill_station.dm

```diff
@@ -162,16 +172,24 @@
 		.["error"] = "Brain not detected. Please consult nearest medical practitioner."
 	else
 		var/list/current_skills = list()
-		for(var/obj/item/skillchip/skill_chip in occupant_brain)
-			current_skills += list(list("name"=skill_chip.skill_name,"icon"=skill_chip.skill_icon,"cost"=skill_chip.slot_cost,"ref"=REF(skill_chip),"active"=(skill_chip in occupant_brain.skillchips)))
+		for(var/obj/item/skillchip/skill_chip in occupant_brain.skillchips)
+			current_skills += list(
+				list(
+					"name" = skill_chip.skill_name,
+					"icon" = skill_chip.skill_icon,
+					"cost" = skill_chip.slot_cost,
+					"ref" = REF(skill_chip),
+					"active" = (skill_chip in occupant_brain.skillchips),
```

Comment:
Can this check ever be false, given the skillchip was taken from this precise list?